app/collins/controllers/actions/asset/PowerStatusAction.scala

```diff
@@ -31,9 +31,12 @@ case class PowerStatusAction(
   }
   override protected def onSuccess(s: CommandResult): ResponseData = {
     logSuccessfulPowerEvent()
-    val status = s.stdout.contains("on") match {
-      case true => "on"
-      case false => "off"
+    val result = s.stdout
+    val status = "error"
+    if (result.contains("Chassis Power is on")) {
+      val status = "on"
+    } else if (result.contains("Chassis Power is off")) {
+      val status = "off"
```

Comment:
I agree with michaeljs1990 unless you have a reason to prefer what you wrote.  I find the suggestion to be closer to the original code and clearer since it introduces fewer variables and syntax